docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Indicators page collects data from enabled threat intelligence feeds and provides a centralized view of indicators, also known as indicators of compromise (IoCs). This topic helps you set up the Indicators page  and explains how to work with IoCs.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators page]
+
+.Requirements
+[sidebar]
+--
+You must have _one_ of the following installed on the hosts you want to monitor:
+
+* *{agent}* - Install a {fleet-guide}/install-fleet-managed-elastic-agent.html[{fleet}-managed {agent}] and ensure the agent's {fleet-guide}/view-elastic-agent-status.html[status] is `Healthy`. Refer to {fleet-guide}/fleet-troubleshooting.html[{fleet} Troubleshooting] if it isn't.
+* *{filebeat}* - Install {filebeat-ref}/filebeat-installation-configuration.html[{filebeat}] version 8.x. Earlier {filebeat} versions are incompatible with ECS and will prevent indicator data from displaying in the Indicators table.
+--
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+Threat intelligence is a research function that helps analyze current and emerging threats, and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.
```

Comment:
The first sentence seems a little muddled to me. From what I remember of the previous draft, threat intel helps _people_ analyze threats and recommend actions, not that threat intel _itself_ analyzes and recommends things. Maybe add people back in, and have them doing most of the action:
```suggestion
Threat intelligence is a research function that helps security professionals analyze current and emerging threats and recommend appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.
```

Reply:
I'm a fan of adding a more tangible actor in the first sentence. The one thing I will note is that "security professionals" is fairly broad. Would you recommend scaling it down to "threat intelligence analysts" @dhru42? Or, is the broader term (security professionals) sufficient?

Reply:
Yeah, I kind of just threw "security professional" in there, no real preference as long as the actor is clearer.

Reply:
I'd recommend the former version where we talk about threat intelligence at the function level because it eludes that threat intelligence analysts can use our functionality, in addition to other analysts interested in accessing the threat intelligence feed output.

Reply:
@dhru42 would the following work?

Threat intelligence is a research function that analyzes current and emerging threats and recommends appropriate actions to strengthen a company's security posture. Threat intelligence requires proactivity to be useful, such as gathering, analyzing, and investigating various threat and vulnerability data sources.